### ISSUE TITLE
Latest Update Error Docker

### ISSUE THREAD

**Issue by razilevin** [microsoft/vscode-remote-release#906]
Please note that  before this update I was able to start the Docker container but had other issues that have been submitted.

- VSCode Version:
```
Version: 1.37.0-insider
Commit: 8627d1e311d1cc1c8fcd1df5688c145a8bf3a6c9
Date: 2019-07-04T17:01:26.869Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 5.0.0-20-generic
```
- Local OS Version:
```
Distributor ID: Ubuntu
Description:    Ubuntu 19.04
Release:        19.04
Codename:       disco
```
- Remote OS Version: Docker FROM node:12.4.0-stretch
- Remote Extension/Connection Type: Docker
```
Name: Remote - Containers
Id: ms-vscode-remote.remote-containers
Description: Open any folder inside (or mounted into) a container and take advantage of Visual Studio Code's full feature set.
Version: 0.66.0
Publisher: Microsoft
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers
```

Steps to Reproduce:

1. Remote Containers - Reopen Folder in Container
2. Error Message 
```
Failed to connect to the remote extension host server (Error: Connection error: Unauthorized client refused.)
```

**Comment by dgitis** [comment]
I'm got this same error when I started up this morning. I rebooted everything and now I don't even get the above error anymore. It looks like VS Code has opened the container fine, but the Bash command prompt is stuck in "Starting."

I wasn't having any issues before and used it fine yesterday.

The steps to reproduce are the same as above.

**VS Code**
Version: 1.37.0-insider (user setup)
Commit: bbfd565b1be08703831d261859955c4fd188b555
Date: 2019-07-05T15:42:24.267Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.17763

**Local OS**
Windows 10
Version	10.0.17763 Build 17763

**Remote OS**
python:3.7-slim-stretch

**Remote Connection Type**
Docker

**Extensions**
Remote - Containers 0.66.0
Remote - SSH 0.44.0
Remote - WSL 0.38.7
Remote Development 0.15.0
